runner/Makefile

```diff
@@ -21,22 +40,47 @@ install-js: clean-js
 	cd $(JS_RUNNER_DIR) \
 	&& npm install .
 
-report: report-js
+install-go:
+	mkdir -p ${GOPATH}/src
+	mkdir -p ${GOPATH}/bin
+	mkdir -p ${GOPATH}/pkg
+	mkdir -p $(GO_PROJECT_DIR)
+	cp $(GO_RUNNER_DIR)/* $(GO_PROJECT_DIR)
+	cd $(GO_PROJECT_DIR) && go get
+	cd $(GO_PROJECT_DIR) && go install
+
+install-java:
+	cd $(JAVA_RUNNER_DIR) \
+	&& ./gradlew build
+
+report: report-js \
+		report-go \
+		report-java
 
 report-js:
 	cd $(JS_RUNNER_DIR) \
 	&& node src/index.js --parser asyncapi-parser --outdir $(REPORTS_JSON) \
 	&& node src/index.js --parser amf-client-js --outdir $(REPORTS_JSON)
 
+report-go:
+	$(GOBIN)/$(GO_PROJECT_NAME) -parser asyncapi-parser -outdir $(REPORTS_JSON)
+
+report-java:
+	cd $(JAVA_RUNNER_DIR) \
+	&& ./gradlew run --args='--parser amf --outdir $(REPORTS_JSON)'
+
 generate-html:
 ifdef tck_branch
 	./node_modules/tck-reporter/bin/tck-reporter.js $(REPORTS_JSON) $(REPORTS_HTML) --repoBranchUrl https://github.com/asyncapi/tck/blob/$(tck_branch)
 else
 	./node_modules/tck-reporter/bin/tck-reporter.js $(REPORTS_JSON) $(REPORTS_HTML) --repoBranchUrl https://github.com/asyncapi/tck/blob/master
 endif
 
-clean: clean-js
+clean: clean-js clean-go
```

Comment:
It may be a stupid question but, don't we need a `clean-java` too?

Reply:
There's no `clean-java` because Java runner doesn't affect the filesystem (visible to the user) during the installation.

The `clean-go` command is there but it's optional because Go runner installs stuff to `/tmp/...`.

Reply:
Oh! Thanks for the clarification. Makes sense!